CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+3. Exception to (2), if the workflow requires some configuration, we should clearly document the requirement in documentation.
+
+### Variables
+
+There are many different types of [variables](https://learning.postman.com/docs/sending-requests/variables/) available in postman. In our playground, we use environment variables and local variables.
+
+#### Environment Variables
+
+Environment variables are scoped to the [environment](https://learning.postman.com/docs/sending-requests/managing-environments/) you selected. We provide two environments in our playground. One is the [demo environment](https://www.postman.com/microstrategysdk/workspace/microstrategy-rest-api/environment/16131298-aac3a1ea-b878-488a-bf80-f4dc5f50a831) environment and the other is a [template](https://www.postman.com/microstrategysdk/workspace/microstrategy-rest-api/environment/16131298-531df55b-93e4-4911-a93d-6d853455e9c5) for a new cloud environment.
+
+In these environments, we provide some long-lived environment variables such as:
+- baseUrl
+- projectId
+- username
+- password
+- loginMode
+Please use these long-lived environment variables over creating your own, whenever possible.
+Note: In the template environment, please change these values to suit your environment.
+
+Environment variables are used to store values that may be used across more than one apis or workflows. For PRs that add/modify content from our collections, you can create/edit these variables through [pre-request scripts and tests](https://learning.postman.com/docs/sending-requests/managing-environments/#setting-environment-variables-from-scripts).
+
+For environment variable naming convention and usage:
+1. We adopt camelCase for variable names.
+  An example can be "`dossierId`" instead of "`DossierID`".
+2. We enforce a "rd_" prefix to runtime generated environment variables(set inside test scripts or pre-request scripts).
+  An example can be "rd_instanceId". This variable is set in the test script of an instance creation api.
+3. We do not have a "rd_" prefix to long-lived environment variables nor using non-runtime environment variables.
+  An example can be "`dossierId`" instead of "`rd_dossierId`" because we plan to use the provided "`dossierId`" in the environment rather than the runtime generated one.
+4. There are special rules that apply to each collection
+  - In the [workflow](https://www.postman.com/microstrategysdk/workspace/microstrategy-rest-api/collection/16131298-9ba9a108-18ee-438e-8f4f-df058f265f1b) collection:
+    - Use runtime environment variables and long-lived environment variables for the path/params/headers/body of api when possible.
+      An example can be we will want to use "`rd_dossierId`" rather than "`dossierId`", because we can not assure the "`dossierId`" will work for every workflow. Hence, we want to generate it at runtime rather than using the existing one in environment.
```

Comment:
The quotes and backtick are redundant I think. Let's keep it simple and just use backtick without quotes.